### PR TITLE
Fix index name in Subread aligner

### PR DESCRIPTION
**Pull request by jkokosar**

### CODE REVIEW

**Comment by kostko** [comment]
Is there a test that catches such errors? If not, perhaps it should be added?

**Comment by jkokosar** [comment]
We test the processors, but in this case the file name we use in the test matched perfectly the "hard-coded" file name for the genome index in the processor code, so the test didn't catch the error.

**Comment by JenkoB** [comment]
Thanks Janez. I see the error, I will be more careful on this.

**Comment by kostko** [comment]
I still think that we should add a test that will catch such errors.

**Comment by dblenkus** [comment]
Do you have any idea how to test this?

It is not good idea to add additional tests, as we would double testing time. Maybe we can generate random names for all files and update db entries after process is done? This would definitely catch all such errors, but is a bit hackish and would (probably) make debugging harder.

**Comment by mstajdohar** [comment]
Test with two genomes with two different names. This is also an opportunity to test strange characters in the genome name. The code should work if the name includes several dots, space...

**Comment by jkokosar** [comment]
Added test as suggested. However, this still doesn't test for the same type of semantic error in other aligner processors.

File: resolwe_bio/tests/processes/test_alignment.py [hunk 1]
```diff
@@ -200,3 +200,10 @@ def test_subread(self):
         inputs = {'genome': genome.id, 'reads': reads_paired.id}
         aligned_reads = self.run_process('alignment-subread', inputs)
         self.assertFile(aligned_reads, 'stats', 'subread_paired_reads_report.txt')
+
+        inputs = {'src': 'my.strange.genome name$.fasta.gz'}
```
Comment (dblenkus) [inline resolwe_bio/tests/processes/test_alignment.py]: You can make the name even stranger, like `Ḡ℮η◎ღℯ` :)

**Pull request merged**